rs/orchestrator/src/boundary_node.rs

```diff
@@ -154,31 +158,39 @@ impl BoundaryNodeManager {
             .as_ref()
             .ok_or_else(|| OrchestratorError::DomainNameMissingError(self.node_id))?;
 
+        const LOG_ANONYMIZATION_CID: &str = "ryjl3-tyaaa-aaaaa-aaaba-cai";
+
         // TODO: Should these values be settable via config?
         let args = vec![
-            format!("--listen-https-port=443"),
-            format!("--tls-hostname={}", domain_name),
-            format!("--tls-cert-path=/var/lib/ic/data/ic-boundary-tls.crt"),
-            format!("--tls-pkey-path=/var/lib/ic/data/ic-boundary-tls.key"),
-            format!("--tls-acme-credentials-path=/var/lib/ic/data"),
-            format!("--http-client-timeout-connect=3s"),
-            format!("--registry-disable-replicator"),
-            format!("--registry-local-store-path=/var/lib/ic/data/ic_registry_local_store"),
-            format!("--obs-log-journald"),
-            format!("--obs-metrics-addr=[::]:9324"),
-            format!("--rate-limit-per-second-per-subnet=1000"),
-            format!("--bouncer-enable"),
-            format!("--bouncer-ratelimit=600"),
-            format!("--bouncer-burst-size=1200"),
             format!("--bouncer-ban-time=5m"),
-            format!("--bouncer-max-buckets=30000"),
             format!("--bouncer-bucket-ttl=1m"),
-            format!("--cache-size=1GB"),
+            format!("--bouncer-burst-size=1200"),
+            format!("--bouncer-enable"),
+            format!("--bouncer-max-buckets=30000"),
+            format!("--bouncer-ratelimit=600"),
             format!("--cache-max-item-size=10MB"),
+            format!("--cache-size=1GB"),
             format!("--cache-ttl=1s"),
-            format!("--shed-system-ewma=0.9"),
+            format!("--http-client-timeout-connect=3s"),
+            format!("--listen-https-port=443"),
+            format!("--obs-log-anonymization-canister-id={LOG_ANONYMIZATION_CID}"),
+            format!("--obs-log-journald"),
+            format!("--obs-metrics-addr=[::]:9324"),
+            format!("--rate-limit-per-second-per-subnet=1000"),
+            format!("--registry-disable-replicator"),
+            format!("--registry-local-store-path=/var/lib/ic/data/ic_registry_local_store"),
             format!("--shed-system-cpu=0.9"),
+            format!("--shed-system-ewma=0.9"),
             format!("--shed-system-memory=0.9"),
+            format!("--tls-acme-credentials-path=/var/lib/ic/data"),
+            format!("--tls-cert-path=/var/lib/ic/data/ic-boundary-tls.crt"),
+            format!("--tls-hostname={domain_name}"),
+            format!("--tls-pkey-path=/var/lib/ic/data/ic-boundary-tls.key"),
+            format!(
+                "--crypto-config={}",
+                serde_json::to_string(&self.crypto_config)
+                    .map_err(OrchestratorError::SerializeCryptoConfigError)?
+            ),
```

Comment:
Have you considered simply passing the path to the config file ([like we do for the replica](https://sourcegraph.com/github.com/dfinity/ic/-/blob/rs/orchestrator/src/upgrade.rs?L477-480)), and do all the parsing on boundary node side? This would also address the `// TODO: Should these values be settable via config` above, and avoid the need for consensus reviews whenever these arguments change.

Reply:
I will defer to your judgement, but personally I am not in favor of providing the downstream process (i.e `ic-boundary`) with a larger than needed configuration. @blind-oracle do you have any thoughts here given that you've mostly owned `ic-boundary`?